examples/acados_python/pendulum_example/minimal_example.py

```diff
@@ -0,0 +1,205 @@
+#
+# Copyright 2019 Gianluca Frison, Dimitris Kouzoupis, Robin Verschueren,
```

Comment:
I guess the pendulum example could be removed since we have this, right?

Reply:
There is a bit less happening in the minimal example.
But yes, I think it makes sense to remove the other pendulum example, and rather add some `example_closed_loop` to show the interaction with the solver.
Some example of this kind will be added soon, so I will remove the old example now.